### DISCUSSION TITLE
License not available

### DISCUSSION THREAD

**Discussion by mbaki** [Security-Onion-Solutions/securityonion#4228] [Unsupported Versions] [answered]
Hi All,

I am trying to Kibana and I get a Request failed with status code 500

Looking at my kibana.log file:

{"type":"log","@timestamp":"2021-05-18T15:29:21+00:00","tags":["error","plugins","security","authentication"],"pid":293,"message":"License is not available, authentication is not possible."}
{"type":"log","@timestamp":"2021-05-18T15:29:21+00:00","tags":["warning","elasticsearch","data"],"pid":293,"message":"Unable to revive connection: https://192.168.2.117:9200/"}
{"type":"response","@timestamp":"2021-05-18T15:29:21+00:00","tags":[],"pid":293,"method":"get","statusCode":503,"req":{"url":"/app/kibana","method":"get","headers":{"user-agent":"curl/7.29.0","host":"localhost:5601","accept":"*/*"},"remoteAddress":"172.17.0.1","userAgent":"curl/7.29.0"},"res":{"statusCode":503,"responseTime":6,"contentLength":9},"message":"GET /app/kibana 503 6ms - 9.0B"}
{"type":"log","@timestamp":"2021-05-18T15:29:21+00:00","tags":["warning","elasticsearch","data"],"pid":293,"message":"No living connections"}
{"type":"log","@timestamp":"2021-05-18T15:29:21+00:00","tags":["warning","plugins","licensing"],"pid":293,"message":"License information could not be obtained from Elasticsearch due to Error: No Living connections error"}

I am running 2.3.50 on a single node.

Thanks
Monah

**Comment by mbaki** [comment]
Elasticsearch failed to start


[root@iaiso elasticsearch]# so-status 

Checking Docker status

    Docker ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]    

Checking container statuses

    so-aptcacherng ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-cortex --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-curator -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-dockerregistry ------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-elastalert ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-elasticsearch ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ]    
    so-filebeat ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-fleet ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-grafana -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-idstools ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-influxdb ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-kibana --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-kratos --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-logstash ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-mysql ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-nginx ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-playbook ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-redis ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-sensoroni ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]    
    so-soc ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]    
    so-soctopus ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-steno ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-strelka-backend ------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]    
    so-strelka-coordinator -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-strelka-filestream --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-strelka-frontend ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-strelka-gatekeeper --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-strelka-manager ------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]    
    so-suricata ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-telegraf ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-thehive -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-thehive-es ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-wazuh ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    
    so-zeek ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]    


----------
          ID: so-elasticsearch-templates
    Function: cmd.run
        Name: /usr/sbin/so-elasticsearch-templates-load
      Result: True
     Comment: Command "/usr/sbin/so-elasticsearch-templates-load" run
     Started: 16:08:28.160169
    Duration: 243124.714 ms
     Changes:   
              ----------
              pid:
                  22758
              retcode:
                  0
              stderr:
              stdout:
                  Waiting for ElasticSearch....................................................................................................................................................................................................................................................
                  Connection attempt timed out.  Unable to connect to ElasticSearch.  
                  Please try: 
                    -checking log(s) in /var/log/elasticsearch/
                    -running 'sudo docker ps' 
                    -running 'sudo so-elastic-restart'
                  
                  Loading templates...
                  so-beats
                  
                  so-common
                  
                  so-firewall
                  
                  so-flow
                  
                  so-ids
                  
                  so-import
                  
                  so-osquery
                  
                  so-ossec
                  
                  so-strelka
                  
                  so-syslog
                  
                  so-zeek

Summary for local
-------------
Succeeded: 32 (changed=3)
Failed:     0
-------------
Total states run:     32
Total run time:  247.226 s

**Comment by gcsrellis** [comment]
I had this issue when I installed from the iso and didnt tab down and use space bar to select OK after accepting the license during setup.  The setup script took my input as not accepting the license.

**Comment by mbaki** [comment, marked as answer]
I had to reinstall and now it's working. Strange.
